tests/test_cli.py

```diff
@@ -66,3 +66,40 @@ def test_cli_harvest_mit_full_legacy_single_success(
         obj={"START_TIME": perf_counter()},
     )
     assert result.exit_code == 0
+
+
+def test_cli_harvest_ogm_no_options_okay_success(runner, mocked_ogm_harvester):
```

Comment:
Could this just be `test_cli_harvest_ogm_no_options_success` without the `okay`?